_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -793,6 +816,17 @@ public static function activate_plugins_permission_check() {
 		return new WP_Error( 'invalid_user_permission_activate_plugins', self::$user_permissions_error_msg, array( 'status' => self::rest_authorization_required_code() ) );
 	}
 
+	/**
+	 * Verify that user can publish posts.
```

Comment:
Nit: here and in the `return` we say just "user", but it'd probably make sense to make it "the current user".